src/foam/u2/stack/Stack.js

```diff
@@ -88,13 +96,20 @@ foam.CLASS({
       return i < 0 ? this.stack_[this.pos + i + 1] : this.stack_[i];
     },
 
-    function push(v, parent, opt_id, opt_hint) {
-      /** opt_id - used to give some unique id to the view being pushed. If it matches the current view then push() ignored. **/
+    function push(/*v, parent, opt_id, opt_hint*/ block) {
+      block = this.StackBlock.create({
```

Comment:
Looks like you reassign the block parameter right away, is that intentional?

Reply:
Only temporary till I can port over all the .push() calls

Reply:
Gotcha 👍